app/views/admin/neighbourhoods/index.html.erb

```diff
@@ -1,7 +1,18 @@
-<%= render_component "admin_index",
-  title: "Neighbourhoods",
-  model: :neighbourhoods,
-  columns: %i[id name district county region],
-  data: @neighbourhoods.where(unit: 'ward'),
-  new_link: new_admin_neighbourhood_path
-%>
+<script>
+var columns = [
+    {"data": "id"},
+    {"data": "name"},
+    {"data": "unit_name"},
+    {"data": "unit_code_key"},
+    {"data": "unit_code_value"},
+  ]
+</script>
+
+<%= render partial: "layouts/admin/datatable", locals: {
+    title: "Neighbourhoods",
+    model: :neighbourhoods,
+    columns: %i[id name unit_name unit_code_key unit_code_value],
+    data: @neighbourhoods.limit(15),
```

Comment:
Again wondering if we want this to be configurable.

Reply:
What do you mean by configurable?

Reply:
What you said on the other one... if there's a dropdown so the user can configure and this is just our default starting point, then all good. I could not tell if it was a fixed value.